Tabs.php

```diff
@@ -290,4 +290,16 @@ public function renderPanes($panes)
     {
         return $this->renderTabContent ? "\n" . Html::tag('div', implode("\n", $panes), ['class' => 'tab-content']) : '';
     }
+    /**
+     * Sets the first visible tab as active
+     */
+    protected function setActiveTab()
+    {
+        foreach ($this->items as $n => $item) {
+            if(!(isset($item['active']) && $item['active'] === false) && ((!isset($item['visible'])) || (isset($item['visible']) && $item['visible'] !== false))){
```

Comment:
The condition looks very complicated. It is possible to move it to separate variable and simplify it a bit?